handlers/handler.ts

```diff
@@ -61,7 +61,7 @@ export interface HandlerDefinition {
 	 *
 	 * Defaults to nodejs14.x if not specified.
 	 */
-	runtime?: 'nodejs14.x' | 'nodejs16.x' | 'nodejs18.x';
+	runtime?: 'nodejs14.x' | 'nodejs16.x' | 'nodejs18.x' | 'nodejs20.x';
```

Comment:
```suggestion
	runtime?: 'nodejs14.x' | 'nodejs16.x' | 'nodejs18.x' | 'nodejs20.x' | string;
```

Reply:
You won't get autocomplete this way. There is a way to do this where you still get autocomplete

Reply:
yeah it's this 
<img width="487" alt="image" src="https://github.com/tailwind/faceteer-cdk/assets/48515765/a1854f69-c8a3-4fda-9e2e-3fb0ef94fc25">

Reply:
making a fix in a new pr since i merged this